library/psa_crypto.c

```diff
@@ -3364,6 +3369,10 @@ psa_status_t psa_sign_hash( mbedtls_svc_key_id_t key,
     status = psa_get_and_lock_key_slot_with_policy( key, &slot,
                                                     PSA_KEY_USAGE_SIGN_HASH,
                                                     alg );
+    /* Return invalid handle as per PSA API 1.0.0 if trying to copy persistent
```

Comment:
_Originally posted by @stevew817 in https://github.com/ARMmbed/mbedtls/pull/4189#discussion_r587568248_
> You're not trying to copy a key here...